libsentrykube/kube.py

```diff
@@ -100,6 +101,21 @@ def _consolidate_variables(
     cluster_name: str = "default",
     external: bool = False,
 ) -> dict:
+    """
```

Comment:
For my own understanding, is there a sentry-kube command which dumps the final values file after all these merges? I thought there was one but I may be misremembering. If you know, mind sharing so that I can refresh my memory.

Reply:
`sentry-kube rendervalues`